src/instructlab/config/init.py

```diff
@@ -108,9 +108,11 @@ def init(
                 model_path,
                 taxonomy_path,
             )
+            if interactive:
```

Comment:
why is this only in the else clause? we want this to happen as well when getting params from env

Reply:
Good catch, will update this